tests/network/docker-compose.yml

```diff
@@ -0,0 +1,140 @@
+networks:
+  ocrd_network_test:
+    name: ocrd_network_test
+    driver: bridge
+    driver_opts:
+      com.docker.network.driver.mtu: 1450
+
+services:
+
+  ocrd_network_mongo_db:
+    image: "mongo"
+    hostname: mongodb-docker-host
+    container_name: ocrd_network_mongo_db
+    networks:
+      - ocrd_network_test
+    ports:
+      - "6701:27017"
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=network_test
+      - MONGO_INITDB_ROOT_PASSWORD=network_test
+    healthcheck:
+      test: echo 'db.runCommand("ping").ok' | mongosh localhost:27017/test --quiet
+      interval: 1s
+      timeout: 3s
+      retries: 30
```

Comment:
Nice, that is a useful pattern we should apply widely.

Reply:
If that works, I prefer using that instead of sleeping here and there to wait. So far I have not seen issues with that health check here and in Operandi.